#pragma once

#include <exception>

using namespace std;

class LoggedSuccessfully : public exception {
};